### PR TITLE
Fix `follow-redirects` library types

### PR DESCRIPTION
**Pull request by fluiddot**
Related to https://github.com/Automattic/studio/pull/410.

## Proposed Changes

- After adding the types of the library `follow-redirects`(https://github.com/Automattic/studio/pull/410/commits/021c0d0316852c505b4497d3c52b70657b800285), I noticed a couple of Typescript errors that are fixed in this PR.

## Testing Instructions

N/A

## Pre-merge Checklist

- [x] Have you checked for TypeScript, React or other console errors?

### CODE REVIEW

File: patches/@types+follow-redirects+1.14.4.patch [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+index 845862e..a7d0587 100644
+--- a/node_modules/@types/follow-redirects/index.d.ts
++++ b/node_modules/@types/follow-redirects/index.d.ts
+@@ -145,3 +145,6 @@ export function wrap<T extends { [key: string]: Scheme<any, any, any> }>(
+ ): {
+     [K in keyof T]: WrappedScheme<T[K]>;
+ };
++
++
++export let maxRedirects: number;
```
Comment (fluiddot) [inline patches/@types+follow-redirects+1.14.4.patch:11]: For some reason, this global variable in the library is not defined in the type definitions.

**Review by sejas — COMMENTED** [review]
I don't see the TS errors. I think Jeroen has fixed it in the original PR:

```
function httpsGet(url: string, callback: (res: IncomingMessage & FollowResponse) => void) {
```

**Comment by fluiddot** [comment]
> I don't see the TS errors. I think Jeroen has fixed it in the original PR:
> 
> ```
> function httpsGet(url: string, callback: (res: IncomingMessage & FollowResponse) => void) {
> ```

@sejas Yep, the TS errors have been fixed in https://github.com/Automattic/studio/pull/410/commits/85e7c4e634a53a780a423d91ccc6372418e8d392. However, in this PR I'm bringing back setting `maxRedirects` globally. If we are ok with the current fixes in the upstream branch, we can close this PR.

**Comment by sejas** [comment]
On my side, I'm fine with the fix on https://github.com/Automattic/studio/commit/85e7c4e634a53a780a423d91ccc6372418e8d392, and I think we are not using the `maxRedirects` anymore. I would close this PR if that's the case.

**Pull request closed without merging**